networkx/algorithms/simple_paths.py

```diff
@@ -68,17 +68,24 @@ def is_simple_path(G, nodes):
     False
 
     """
+    assert isinstance(nodes, list), "Object passed as `nodes` must be a list."
+
     # The empty list is not a valid path. Could also return
     # NetworkXPointlessConcept here.
-    if len(nodes) == 0:
+    if not nodes:
+        return False
```

Comment:
Similarly, I'm -1 on this too. It makes sense with the above, but not by itself.